### ISSUE TITLE
In case of blocked scatter default node, scatter don't throw any exception when "send" a transaction

### ISSUE THREAD

**Issue by qpIlIpp** [GetScatter/ScatterDesktop#312]
We have default scatter node ( nodes.get-scatter.com ) blocked by Roskomnadzor (Federal Service for Supervision of Communications, Information Technology and Mass Media) in Russia. 
https://isitblockedinrussia.com/?host=nodes.get-scatter.com
I added an account to scatter with VPN using and turned my VPN off.
In the case of web extension, it was enough to use it.
Now scatter is able to log in, but when it received an `eos.transaction` it does nothing.
Scatter should throw an error, it will let my front-end make a message "scatter node is unavailable, add another one node".
I have tested it on different dapps, even though bloks.io/wallet doesn't work.

**Comment by qpIlIpp** [comment]
20 days. Any updates?

**Comment by nsjames** [comment]
Cleaning tickets for Scatter 11 testing release (makes this ticket no longer relevant)

**Comment by qpIlIpp** [comment]
@nsjames I've check. Problem is still there. No errors in case of network unavailability. 
![image](https://user-images.githubusercontent.com/32383682/60260361-d71cb080-98f2-11e9-924d-24992ac94320.png)

**Comment by nsjames** [comment]
These should throw errors in the app saying it can't connect to the API, but it won't give you a resolution because it could be various things. We've yet to work out a good system for explaining and displaying possible fixes for known issues.
